model/prelude.sail

```diff
@@ -191,14 +191,15 @@ function rotatel (v, n) =
 overload operator >>> = {rotate_bits_right, rotater}
 overload operator <<< = {rotate_bits_left, rotatel}
 
-function reverse_bits_in_byte (xs : bits(8)) -> bits(8) = {
-  var ys : bits(8) = zeros();
-  foreach (i from 0 to 7)
-    ys[i] = xs[7-i];
+val reverse_bits : forall 'n, 'n > 0. bits('n) -> bits('n)
```

Comment:
I think you don't need the 'n > 0 with the latest Sail compiler. IIRC it implicitly adds it since 'n is used in bits().